libs/sdk-ui-kit/api/sdk-ui-kit.api.md

```diff
@@ -2581,12 +2601,26 @@ export interface IShareGranteeBaseProps {
     // (undocumented)
     isLoading: boolean;
     // (undocumented)
+    isLocked: boolean;
```

Comment:
Consider wrap all this flag into one object "accessControl"  or similar and pass them as one prop.

Reply:
I wrapped shared object related props to a single object.